tests/system/cases/service_scale/main.tf

```diff
@@ -6,9 +6,11 @@ provider "layer0" {
 
 resource "layer0_environment" "ss" {
   name = "ss"
+  scale = 3
```

Comment:
Scaling the environment to 3 doesn't seem necessary?

Reply:
Correct! That's a typo. :)

✅